tests/test_cases/test_discovery/test_vhdl_indexed_name.py

```diff
@@ -26,12 +26,20 @@
 import cocotb
 
 
-@cocotb.test()
+@cocotb.test(skip=True)
```

Comment:
Skipped for a reason?

Reply:
It doesn't actually test anything. In fact I'll do some more refactoring